src/pages/Categories/CategoriesBeta.vue

```diff
@@ -15,6 +15,7 @@
 			<kv-page-container>
 				<kv-grid>
 					<loan-spotlight
+						data-testid="all-categories-loan-spotlight"
```

Comment:
Please add the `data-testid` inside of the components, this will avoid duplicating them here and they will be available for future uses of the component